app/frontend/tests/unit/components/forms/manage/DocumentTemplate.spec.js

```diff
@@ -497,6 +497,9 @@ describe('DocumentTemplate.vue', () => {
     });
 
     await flushPromises();
+    windowOpenSpy.mockReset();
+    appendChildSpy.mockReset();
+    addNotificationSpy.mockReset();
```

Comment:
I think something in the `mount` was calling these spies. Since we're not testing the mount, they should be cleared afterwards (tests were failing)